src/analyze/analyze.c

```diff
@@ -2145,6 +2177,49 @@ static int do_condition(int argc, char *argv[], void *userdata) {
 }
 
 static int do_verify(int argc, char *argv[], void *userdata) {
+        int r, i = 1;
+        char **filename;
+        char *orig, *base;
+        _cleanup_free_ char *alias = NULL;
+        _cleanup_(rm_rf_and_freep) char *tempdir = NULL;
+        bool use_alias = false;
+
+        STRV_FOREACH(filename, strv_skip(argv, 1))
+                if (strchr(*filename, ':')) {
+                        use_alias = true;
+                        break;
+                }
+
+        if (use_alias) {
+                r = setup_working_dir(&tempdir);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to setup working directory");
+
+                STRV_FOREACH(filename, strv_skip(argv, 1)) {
+                        use_alias = strchr(*filename, ':');
+                        if (use_alias) {
+                                extract_first_word((const char **)filename, &orig, ":", 0);
+                                base = *filename;
+                        } else { /* symlink to keep everything in one path */
+                                base = basename(*filename);
+                                orig = argv[i];
+                        }
+
+                        alias = path_join(tempdir, base);
+                        if (!alias)
+                                return log_oom();
+
+                        r = symlink(orig, alias);
+                        if (r < 0)
+                                return log_error_errno(errno, "Couldn't create symlink %s %s %s",
```

Comment:
`orig` leaks here

Reply:
How does this leak? I call 
```
if (use_alias)
        free(orig);
```

Reply:
if there's an error here, it never gets to that `free`

Reply:
Right, makes sense